kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/SectionSidePanel.vue

```diff
@@ -105,7 +105,10 @@
     },
     methods: {
       handleClosePanel() {
-        if (this.workingResourcePool.length > this.activeResourcePool.length) {
+        if (
+          this.workingResourcePool &&
```

Comment:
For some reason `workingResourcePool` was undefined here and it didn't let me move forward